drv/mr_timer_hf/mr_timer_hf.c

```diff
@@ -19,6 +19,7 @@
 
 //=========================== define ===========================================
 
+#define TIMER_IRQ_PRIORITY (2U)  ///< Timer interrupt priority
```

Comment:
Consider adding a comment explaining why we are changing the priority to 2 (for the specific case of UART), and that it is a workaround.